asset-transfer-basic/chaincode-external/chaincode.env

```diff
@@ -6,3 +6,18 @@ CHAINCODE_SERVER_ADDRESS=asset-transfer-basic.org1.example.com:9999
 # on install. The `peer lifecycle chaincode queryinstalled` command can be
 # used to get the ID after install if required
 CHAINCODE_ID=basic_1.0:0262396ccaffaa2174bc09f750f742319c4f14d60b16334d2c8921b6842c090c
+
+# Optional parameters that will be used for TLS connection between peer node
+# and the chaincode.
+# Default value is true, set this to false to use TLS connection
+# CHAINCODE_TLS_DISABLED=false
```

Comment:
This hurts my brain. :-) Why did you choose to make it that way rather than the other way around:

```
# Default value is false, set this to true to use TLS connection
# CHAINCODE_TLS_ENABLED=false
```

Reply:
Ok, I understand. I think what hurts my head is also that it reads that the default value is true when the next line shows false. I understand it is commented out but this is confusing. So, I suggest:
```suggestion
# TLS is disabled by default, uncomment the following line to enable TLS connection
# CHAINCODE_TLS_DISABLED=false
```

Reply:
Got it! This is great idea. Changing the comment line will make it more readable.